lfsapi/ssh.go

```diff
@@ -59,18 +59,27 @@ type sshAuthResponse struct {
 	Message   string            `json:"-"`
 	Href      string            `json:"href"`
 	Header    map[string]string `json:"header"`
-	ExpiresAt time.Time         `json:"expires_at"`
-	ExpiresIn int               `json:"expires_in"`
+	ExpiresAt *time.Time        `json:"expires_at"`
```

Comment:
I'd like to add getters for these two pointers and handle the default values in those for cleanliness but I'd like to get another opinion before I do that. Alternatively I could check them in the same way that tools.IsExpiredAtOrIn does and simply not use the value at all if its nil.

Reply:
> Alternatively I could check them in the same way that tools.IsExpiredAtOrIn does and simply not use the value at all if its nil.

I think that I prefer this one, since only public values are able to be seen by `encoding/json`, and it would be good to reduce the number of 'entries' into a value (either by calling a getter or accessing the field directly).